bootstrap.sh

```diff
@@ -1 +1,30 @@
 #!/bin/bash
+
+echo "Creating namespace..."
+kubectl apply -f ./.infrastructure/namespace.yml
+
+echo "Applying ConfigMap..."
+kubectl apply -f ./.infrastructure/confgiMap.yml
```

Comment:
There is a typo in the file path: `confgiMap.yml` should be `configMap.yml`. Ensure the file name matches the actual file in your infrastructure directory.